functions/package.json

```diff
@@ -2,12 +2,20 @@
   "name": "functions",
   "description": "Cloud Functions for Firebase",
   "scripts": {
-    "lint": "eslint .",
-    "serve": "firebase emulators:start --only functions",
-    "shell": "firebase functions:shell",
+    "config:get": "npx firebase functions:config:get",
+    "config:test": "npx firebase use test",
+    "delete:poll": "npx firebase functions:delete poll",
+    "delete:poll:force": "npx firebase functions:delete poll --force",
+    "deploy": "npx firebase deploy --only functions",
+    "deploy:poll": "npx firebase deploy --only functions:poll",
+    "lint": "npx eslint .",
+    "lint:fix": "npx eslint . --fix",
+    "logs": "npx firebase functions:log",
+    "logs:poll:vols": "npx firebase functions:log --only poll-volunteers",
+    "reboot:poll": "run-s delete:poll:force deploy:poll",
+    "serve": "npx firebase emulators:start --only functions",
+    "shell": "npx firebase functions:shell",
```

Comment:
changes are:
- use `npx` to invoke local `devDependencies` installs properly when relevant cf #33 
- take a bunch of firebase-y things and put them in scripts so people don't have to continuously look them up until they turn into muscle memory

Reply:
note that there is some overlap between the `scripts` in #33